iceoryx2/src/service/static_config/message_type_details.rs

```diff
@@ -156,19 +177,19 @@ mod tests {
         let expected = MessageTypeDetails{
             header:  TypeDetail{
                 variant: TypeVariant::FixedSize,
-                type_name: "i32".to_string(),
+                type_name: "i32".try_into().unwrap(),
```

Comment:
@elfenpiff Thoughts on use of `unwrap()` here ? It seemed like overkill to use `fatal_panic!()` for these ...

Reply:
This is completely alright. One sees directly in the code that this is fine!